packages/hdwallet-metamask-multichain/src/adapter.ts

```diff
@@ -0,0 +1,62 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+import { enableShapeShiftSnap, shapeShiftSnapInstalled } from "@shapeshiftoss/metamask-snaps-adapter";
+import { createStore } from "mipd";
+
+import { SNAP_ID } from "./common";
+import { MetaMaskShapeShiftMultiChainHDWallet } from "./shapeshift-multichain";
```

Comment:
we should update class name to match while we are at it

Reply:
a9fa9f5079c02d12f9800e9e182b88a5f8ffa141 (don't mind the diff, needed actual bimp for `integration` not to be borked... which I also added missing package.json entry for [there](https://github.com/shapeshift/hdwallet/commit/a9fa9f5079c02d12f9800e9e182b88a5f8ffa141#diff-cd67598767aad4b2e7c64086173ce0c83943deb401b2a81f8e3d835d8141cbc9R18), was shadowed previously)

Reply:
Actually: https://github.com/shapeshift/hdwallet/pull/689#discussion_r1787469657